packages/profile-sync-controller/package.json

```diff
@@ -102,8 +102,8 @@
   "dependencies": {
     "@metamask/base-controller": "^7.0.2",
     "@metamask/keyring-api": "^10.1.0",
-    "@metamask/keyring-controller": "^19.0.0",
-    "@metamask/network-controller": "^22.0.2",
+    "@metamask/keyring-controller": "^19.0.1",
+    "@metamask/network-controller": "^22.1.0",
```

Comment:
These should be `devDependencies` instead. Unrelated to the release though, we can fix this after